cmd/preflight/cli/stdout_results.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+
+	analyzerunner "github.com/replicatedhq/troubleshoot/pkg/analyze"
+)
+
+var failed = false
+
+func showStdoutResults(preflightName string, analyzeResults []*analyzerunner.AnalyzeResult) error {
+	fmt.Printf("\n=== TEST   %s\n", preflightName)
+	for _, analyzeResult := range analyzeResults {
+		fmt.Printf("=== RUN:   %s\n", analyzeResult.Title)
+	}
+	for _, analyzeResult := range analyzeResults {
+		outputResult(analyzeResult)
```

Comment:
`outputResult` should return the value that is currently stored in the global `failed` variable.   The the global variable should be removed.